### PR TITLE
retty.me

### LINKED ISSUE #108093: retty.me
### Issue URL (Ads)
 [https://retty.me/area/PRE13/ARE1/SUB101/100001455698/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fretty.me%2Farea%2FPRE13%2FARE1%2FSUB101%2F100001455698%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #108093 

<details>
<summary>MEMO</summary>

! https://retty.me/
! MOBILE
! Ad
retty.me##div[class*="-ad"]
!
! https://retty.me/area/PRE13/ARE1/SUB101/100001455698/
! MOBILE
! app
retty.me##.app-store-banner-wrap
retty.me##.js-app-store-banner
!
! https://retty.me/area/PRE28/ARE98/SUB9801/100001616393/
! MOBILE
! Ad
retty.me##div[style^="margin: 30px auto 0;"]
! social (tap "送る")
retty.me##.share-dialog__spacer

</details>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -1154,6 +1154,9 @@ natuyuki.com##.cta
 i486.mods.jp##table[style$="width: 300px;"]
 rocomotion.net###maintitle_ad
 retty.me##.dfp-banners
+retty.me##div[style^="margin: 30px auto 0;"]
+retty.me##.middle-ad-section
+retty.me##.small-ad-content
```
Comment (gork7777) [inline JapaneseFilter/sections/specific.txt:1159]: What about change that two rules to  `##div[class*="-ad"]` rule ? You can add it for: `kai-you.net##div[class*="-ad"]`

Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:1159]: I see now.

Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt]: @gork7777  thank you for good advice

Comment (timothy-vas) [reply JapaneseFilter/sections/specific.txt]: @gork7777 So are you the one that decides how to fix reports? There is nothing here that concerns you. And stop sticking your nose into pull requests.
Developers can handle this

**Review by zloyden — APPROVED** [review]

**Pull request merged**